src/main/resources/application.yaml

```diff
@@ -1,4 +1,17 @@
 server:
   port: 8080
   servlet:
-    context-path: /art-gallery
\ No newline at end of file
+    context-path: /art-gallery
+
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: "jdbc:postgresql://localhost:5432/postgres"
+    username: "postgres"
+    password: "mysecretpassword"
+    name: "postgres"
+  jooq:
+    sql-dialect: POSTGRES
+  liquibase:
+    change-log: classpath:db/changelog.xml
+    drop-first: true
```

Comment:
drop-first nigdy nie powinno pojawić się w produkcyjnym kodzie

Reply:
nie wiedziałam gdzie to zmienić, ale teraz już wiem dzięki komentarzowi, mersi bełkepu C: